packages/web3-utils/src/json_rpc.ts

```diff
@@ -28,9 +28,71 @@ import {
 	JsonRpcBatchResponse,
 	JsonRpcSubscriptionResult,
 } from 'web3-types';
-
+import {
+	ParseError,
+	InvalidRequestError,
+	MethodNotFoundError,
+	InvalidParamsError,
+	InternalError,
+	InvalidInputError,
+	ResourcesNotFoundError,
+	ResourceUnavailableError,
+	TransactionRejectedError,
+	MethodNotSupported,
+	LimitExceededError,
+	VersionNotSupportedError,
+	RpcError,
+	rpcErrorValues,
+} from 'web3-errors';
 import { uuidV4 } from './uuid';
 
+export const getRpcError = (code: number) => {
+	const codeStr = code.toString();
```

Comment:
if we use map as shared above we can avoid these error codes duplication as well and can directly query that map where required as:
`rpcErrorsMap.get(-32700) ` // in web3-core request manager directly